src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/services/servicebus/ServiceBusHelperTest.java

```diff
@@ -196,6 +205,24 @@ private void mockEnvelopeData() {
         when(scannableItem2.getDocumentSubtype()).thenReturn(DocumentSubtype.SSCS1);
         when(scannableItem2.getScanningDate()).thenReturn(Instant.now());
         when(scannableItem2.getOcrData()).thenReturn(null);
+
+        when(payment1.getDocumentControlNumber()).thenReturn("dcn1");
+        when(payment2.getDocumentControlNumber()).thenReturn("dcn2");
+    }
+
+    private void assertMessageContainsRightPayments(JsonNode jsonMessage) {
+        JsonNode payments = jsonMessage.get("payments");
+        assertThat(payments.isArray()).isTrue();
+        assertThat(payments.size()).isEqualTo(2);
+        assertSamePaymentData(payments.get(0), payment1);
```

Comment:
may be something like below would be easy
```
assertThat(payments).extracting("document_control_number")
                               .contains("dcn1","dcn2")
```

Reply:
how about
```
        assertThat(payments)
            .extracting(item -> item.get("document_control_number").asText())
            .containsExactly(
                payment1.getDocumentControlNumber(),
                payment2.getDocumentControlNumber()
            );
``` ?